pkg/repository/udmrepo/kopialib/backend/common.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package backend
+
+import (
+	"context"
+	"time"
+
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/repo/blob"
+	"github.com/kopia/kopia/repo/blob/throttling"
+	"github.com/kopia/kopia/repo/content"
+	"github.com/kopia/kopia/repo/encryption"
+	"github.com/kopia/kopia/repo/hashing"
+	"github.com/kopia/kopia/repo/object"
+	"github.com/kopia/kopia/repo/splitter"
+
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo"
+)
+
+const (
+	maxDataCacheMB         = 2000
+	maxMetadataCacheMB     = 2000
+	maxCacheDurationSecond = 30
+)
```

Comment:
What is the reference setting for these three theoretical values?

Reply:
This is according to Kopia's default value:
maxDataCacheMB: 5G
maxMetadataCacheMB: 5G
maxCacheDurationSecond: 30s

The default values are somehow aggressive for docker env, where the default root disk size is 10G, so I changed the size values to 2G respectively. Let's see it in the performance testing, if we find problem, we change it accordingly.